runtime/lua/vim/treesitter.lua

```diff
@@ -1,4 +1,6 @@
 local a = vim.api
+local query = require'vim.treesitter.query'
+local lang = require'vim.treesitter.language'
```

Comment:
I don't really like having `lang` be the import from `language`.

1. it's confusing to shorten the module
2. it seems too easy to overwrite. Perhaps some more unique name would be better.